tasks/computer-vision/image-classification/README.md

```diff
@@ -1 +1,10 @@
-# Image Classification
+# Image Classification
+
+
+Developed an Residual Attention Network where the main branch is based off ResNet18 architecture and the mask branch is a AutoEncoder (U-Net).
+
+Tested on MRI image classification.
+
+The Hyperparameters file can be changed for training and to run type:
+
+python main.py

```

Comment:
please add `mlflow run .` example. 
Optional: Would it be possible to add more examples or details about the parameters?